pkg/policy/l4.go

```diff
@@ -1229,17 +1243,25 @@ func (l4 *L4Policy) GetModel() *models.L4Policy {
 
 	ingress := []*models.PolicyRule{}
 	for _, v := range l4.Ingress {
+		derivedFrom := labels.LabelArrayList{}
+		for _, rules := range v.RuleOrigin {
+			derivedFrom = append(derivedFrom, rules...)
```

Comment:
Does the fact that we iterate over a map here change the order of labels in `derivedFrom`? Or was it already not a stable order before?

Reply:
good catch: It was a stable oder before; sorted even.